libs/coin-modules/coin-cardano/src/cli-transaction.ts

```diff
@@ -31,7 +30,7 @@ function inferAccounts(account: Account, opts: Record<string, any>): Account[] |
     return accounts;
   }
 
-  return opts.token.map(token => {
+  return opts.token.map((token: any) => {
```

Comment:
Is any necessary ? I don't think it should

Reply:
Typescript settings in CoinModule raise an error if there is an implicite type `any`. To fix it, I just explicit the type.